agenta-web/src/components/AppSelector/Welcome.tsx

```diff
@@ -81,85 +75,89 @@ interface Props {
-                        please follow our{" "}
-                        <a href="https://docs.agenta.ai/installation" target="_blank">
-                            installation guide
-                        </a>
-                        .
-                    </p>
-                )}
-                {isDemo && (
-                    <p>
-                        Important Note: You are using the demo version of Agenta. Don't forget to
-                        regularly back up your data!
-                        <br />
-                        If you're considering integrating Agenta into your workflow, feel free to{" "}
-                        <a
-                            href="https://join.slack.com/t/agenta-hq/shared_invite/zt-1zsafop5i-Y7~ZySbhRZvKVPV5DO_7IA"
-                            target="_blank"
+                    <div className={classes.steps}>
+                        <h3>Workflow Essentials</h3>
+                        <Text>- Start from a template</Text>
+                        <Text>- Compare prompts and models</Text>
+                        <Text>- Create testsets</Text>
+                        <Text>- Evaluate outputs</Text>
+                        <Text>- Deploy in one click</Text>
+                    </div>
+
+                    <div className={classes.btnContainer}>
+                        <Button size="large" type="primary" onClick={onCreateAppClick}>
+                            Start
+                        </Button>
+                    </div>
+                </div>
+                <div className={classes.container}>
+                    <div className={classes.title}>
+                        <h1>Build complex apps</h1>
+                        <Tag
+                            className={classes.tag}
+                            icon={<ClockCircleOutlined />}
+                            color={appTheme === "dark" ? "warning" : "green"}
                         >
-                            {" "}
-                            join us on Slack{" "}
-                        </a>{" "}
-                        or{" "}
-                        <a href="https://cal.com/mahmoud-mabrouk-ogzgey/demo" target="_blank">
-                            schedule an onboarding session
-                        </a>
-                        .
-                    </p>
-                )}
+                            12-15 mins
+                        </Tag>
+                    </div>
 
-                <Button type="primary" onClick={onCreateAppClick}>
-                    Create New App
-                </Button>
-            </div>
-        </div>
+                    <img
+                        src="/assets/complex-img.png"
+                        alt="Complex build Image"
+                        className={classes.img}
+                    />
+
+                    <div className={classes.steps}>
+                        <h3>Workflow Essentials</h3>
+                        <Text>- Start from code</Text>
```

Comment:
Same here, would advise to wrap in ul and li elements. Also instead of hardcoding `-` everywhere could you please add this style in the css to replace the manual dashes.

```css
ul {
    list-style-type: '- ';
}
```